ui/package.json

```diff
@@ -183,7 +183,7 @@
     "node": "10.* || >= 12"
   },
   "ember": {
-    "edition": "octane"
+    "edition": "recommended"
```

Comment:
that's interesting.  do you know why this had to change?

Reply:
Oh good catch, I changed this when I was wrestling with linting changes. I'll change it back and make sure linting is still good 👍